docs/source/docs/index.rst

```diff
@@ -136,89 +150,99 @@ Read the research:
    ../overview
    ../getting-started/installation
    ../getting-started/quickstart
-   ../examples/interactive-development
-   ../getting-started/tutorial
+   Example: AI Training <../getting-started/tutorial>
    ../sky-computing
 
-
 .. toctree::
    :hidden:
    :maxdepth: 1
-   :caption: Running Jobs
+   :caption: Clusters
 
-   ../examples/managed-jobs
-   ../reference/job-queue
+   Start a Cluster <../examples/interactive-development>
    ../examples/auto-failover
-   ../running-jobs/distributed-jobs
-   ../running-jobs/many-jobs
+   ../reference/auto-stop
 
 .. toctree::
    :hidden:
    :maxdepth: 1
-   :caption: Reserved & Existing Clusters
+   :caption: Jobs
+
+   Cluster Jobs <../reference/job-queue>
+   ../examples/managed-jobs
+   Multi-Node Jobs <../running-jobs/distributed-jobs>
+   Many Parallel Jobs <../running-jobs/many-jobs>
 
-   ../reservations/reservations
-   Using Existing Machines <../reservations/existing-machines>
-   ../reference/kubernetes/index
 
 .. toctree::
    :hidden:
    :maxdepth: 1
-   :caption: SkyServe: Model Serving
+   :caption: Model Serving
 
-   ../serving/sky-serve
+   Getting Started <../serving/sky-serve>
    ../serving/user-guides
```

Comment:
When merging with master, may need to move the HTTP page under user guides

Reply:
Done.